.node-version

```diff
@@ -0,0 +1 @@
+5
```

Comment:
There's also usually a setting in your text editor to ensure a new line character at the end of every file.